var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -31,7 +31,7 @@
 from spack import *
 
 
-class Perl(Package):
+class Perl(Package): # Perl does not use Autotools, it should subclass Package
```

Comment:
Can you add one more space before this comment? It is causing the flake8 tests to fail.